app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e
+
+USAGE="
+Usage: $0 [etcd flags]
+
+Wrapper for launching etcd via rkt.
+
+The version of etcd to be used can be set through ETCD_VERSION. The version
+must be set to one of the image tags published here:
+
+  https://quay.io/repository/coreos/etcd?tab=tags
+
+Alternatively, the keywords 'alpha', 'beta', and 'stable' may be used to specify
+the version. If no version is provided, the CoreOS update group is used.
+"
+
+case ${1} in
+	-h | --help)
+		echo "${USAGE}"
+		exit 0
+		;;
+esac
+
+source /usr/share/coreos/update.conf
+source /etc/coreos/update.conf
+
+ETCD_VERSION="${ETCD_VERSION:-${GROUP:-stable}}"
+ETCD_ACI="${ETCD_ACI:-quay.io/coreos/etcd}"
+
+case ${ETCD_VERSION} in
+	stable)
+		ETCD_VERSION=v2.3.5
```

Comment:
This isn't v2.3.6 because I get the following error:

```
run: requested value for label "version": "v2.3.6" differs from fetched aci label value: "latest"
```

It looks like we end up downloading "latest" instead of "v2.3.6" even though they are the same thing.